### ISSUE TITLE
Site is slow maybe due https://asset.launch.mojohelpdesk.com/javascripts/form_widget.js 

### ISSUE THREAD

**Issue by eriol** [circuits/circuits.github.io#3]
Hello,
I did not investigate deeply, but it seems that some dependency stuff for the site doesn't exists anymore:

```
❯ curl https://asset.launch.mojohelpdesk.com/javascripts/form_widget.js
curl: (28) Failed to connect to asset.launch.mojohelpdesk.com port 443 after 129589 ms: Couldn't connect to server
```
This slow down the site.

Google lighthouse give a speed index of 15.4 sec:
![image](https://user-images.githubusercontent.com/212279/208291660-1c495237-58ca-4bf8-99ca-e43fbf663eed.png)

**Comment by prologic** [comment]
@spaceone We can probably clean up the website here and remove a bunch of "cruft" like this 🤔

**Comment by spaceone** [comment]
yes, but I don't find the source code which generates that mojohelpdesk link.
I also don't have access rights to push the latest version. I once updated the IRC links, etc. and expected a pipeline to build the new website but that's not the case. There is a Makefile to do that.

**Comment by prologic** [comment]
@spaceone You should be an owner of the org and therefore have full access to everything no? 🤔

**Comment by spaceone** [comment]
> You should be an owner of the org and therefore have full access to everything no? thinking
Yes, but not for external services like mojohelpdesk.com and somehow also not TravisCI.
The website is just an git-repo, so that should work.

The first push of the website was done by TravisCI.

When I use the latest pelican and upgrade the website the template is lost: See 6d59b573410b2bdde3bc6f7222162e50680ece12 

I will investigate later. For now I just dropped the broken script access.

**Comment by spaceone** [comment]
I am also missing access to readthedocs.org

**Comment by prologic** [comment]
> I am also missing access to readthedocs.org

I'll look into this later today 👌

**Comment by prologic** [comment]
@spaceone You should have maintainer access to the readthedocs.org project for circuits now 👌
